packages/vendure-plugin-invoices/src/services/invoice.service.ts

```diff
@@ -243,7 +244,20 @@ export class InvoiceService implements OnModuleInit, OnApplicationBootstrap {
       path: tmpFilePath,
       type: '',
     };
-    await pdf.create(document, options);
+
+    if (process.env.NODE_ENV === 'test') {
+      pdf.create(document, options).catch((error: any) => {
+        Logger.error(JSON.stringify(error), loggerCtx);
+      });
+    } else {
+      try {
+        await pdf.create(document, options);
+      } catch (e) {
+        Logger.error(JSON.stringify(e), loggerCtx);
+        throw e;
```

Comment:
The try/catch is a good addition, but we shouldn't have any NODE_ENV===test checks in production code. It's good practice to make sure code behaves the same way in production as it would in test. 

Temporary log statements are fine, just remove them before creating a PR ;-) 

If you keep the catch+logging, could you make it a human readable log? Something like `Could not create PDF` + error message.